samples/custom-login/public/index.html

```diff
@@ -0,0 +1,192 @@
+<html>
+  <head>
+    <!-- https://global.oktacdn.com/okta-auth-js/4.0.0/okta-auth-js.polyfill.js -->
+    <script src="/okta-auth-js.polyfill.js" type="text/javascript"></script>
+
+    <!-- https://global.oktacdn.com/okta-auth-js/4.0.0/okta-auth-js.min.js -->
+    <script src="/okta-auth-js.min.js" type="text/javascript"></script>
+
+    <!-- signin widget -->
```

Comment:
I would suggest a class on these links so we can script something to easily update to latest minor version.

Reply:
Samples are now generated from templates, so the version will update automatically based on what is installed in node_modules